CHANGELOG.md

```diff
@@ -4,36 +4,15 @@ All notable changes to this project will be documented in this file. Dates are d
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
-#### [v1.5.3](https://github.com/microsoft/tabster/compare/v1.5.2...v1.5.3)
```

Comment:
revert the changelog changes please, we still want them. We just want to add a changelog entry that mentions the versions being reverted

Reply:
Theoretically, it should be automatically regenerated. Let's try.